daemon/wait_handle/wait_handle.go

```diff
@@ -81,8 +82,16 @@ func Call() {
 
 	healthCheckClient := &http.Client{
 		Timeout: constants.HC_Timeout * time.Second,
+		Transport: &http.Transport{
+			TLSClientConfig: &tls.Config{
+				InsecureSkipVerify: true,
```

Comment:
Yes but also because porterd is calling localhost which doesn't match the domain so on a https redirect the call would fail